app/models/pydantic/authentication.py

```diff
@@ -20,7 +20,8 @@ class User(StrictBaseModel):
     email: EmailStr
     createdAt: datetime
     role: str
-    applications: List[str]
+    provider: str
+    providerId: str
```

Comment:
I don't see providerId listed in https://resource-watch.github.io/doc-api/reference.html#get-lt-base-api-url-gt-auth-check-logged .  Is that documentation out of date?  If it is not being returned by 'check-logged', can you put a comment in on why you're adding it?

Also, it looks like 'updatedAt' is returned by 'checked-logged'.  Do you want to include that for completeness?

Reply:
I had one unpushed commit that makes providerId optional as it's only present when `provider` represents third party auth (facebook, google, etc). Thanks for catching it.
Also `updatedAt` is not present in the actual response.
The documentation looks incorrect/incomplete in both cases.